README.md

```diff
@@ -103,6 +105,97 @@ To do that - add `[New snapshots]` to the pull request title. Otherwise the snap
 
 We also are having [design system documentation](components/README.md).
 
+## 🧐 Firebase Analytics
```

Comment:
suggestion:
1. Let's move to a different file so our main README file doesn't contain too much data. e.g. `[Analytics Documentation](docs/analytics.md)`

2. It would be good to have images in the repo, so we can do normal code-review process. So move images to GitHub repo `docs/analytics-imgs/users-in-last-30-minues.png`

3. "Events Names" documentation, should have more information about every specific event. You can look at [button_click](https://www.notion.so/appunite/button_click-76d9bd6c4a5647e5bb09061e64f11e0f?pvs=4). We should know details when something is tracked, what parameters are required. Some examples, etc.

4. Afaik, it would be good if every screen would have its own custom README page, e.g. `docs/anlaytics/page/reviewers-screen.md`
5. As a start section of a given screen, it's good to add a screenshot of that page, so it's easier to know exactly what screen we're referring in the following events.